scripts/s3.py

```diff
@@ -174,6 +178,8 @@ def main():
 
     s3 = boto3.client("s3", aws_access_key_id=aws_key, aws_secret_access_key=aws_secret)
 
+    print(json.dumps(data))
```

Comment:
Print the entire data snapshot to console (we need this data more often than not when debugging/testing)